runtime/v2/builder.go

```diff
@@ -116,6 +121,22 @@ func (a *AppBuilder[T]) Build(opts ...AppBuilderOption[T]) (*App[T], error) {
 
 	a.app.stf = stf
 
+	v := a.viper
+	home := v.GetString(FlagHome)
+	scRawDb, err := db.NewGoLevelDB("application", filepath.Join(home, "data"), nil)
```

Comment:
so store/v2 is goleveldb only @kocubinski?

Reply:
we can handle multiple dbs by `db-backend` in app.toml

Reply:
That's for v1, but maybe v2 doesn't support multiple backend? Wanted to confirm, otherwise this being hardcoded in runtime isn't great (but easily solvable with an appbuilder options). Let's wait for an answer

Reply:
store v2 has support for pebbledb as well